<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimWindow</class>
 <widget class="QWidget" name="SimWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Симуляция</string>
  </property>
  <widget class="QTextBrowser" name="DayCounterText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;День 1 из 7&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>1031</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Заказы магазинов</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTreeWidget" name="ShopsOrder">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Магазин</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Продукт</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Кол-во</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Уцененный</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Цена за продукт</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>Магазин 1</string>
       </property>
       <property name="text">
        <string>Сахар</string>
       </property>
       <property name="text">
        <string>240кг</string>
       </property>
       <property name="text">
        <string>Да</string>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Магазин 1</string>
       </property>
       <property name="text">
        <string>Клубника</string>
       </property>
       <property name="text">
        <string>10кг</string>
       </property>
       <property name="text">
        <string>Нет</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Заказы поставщику</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTreeWidget" name="ProviderOrders">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Номер заказа</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дней для выполнения</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Продукт</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Кол-во упаковок</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Товары на складе</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTreeWidget" name="StorageItems">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Продукт</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>День поставки</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>День выхода срока годности</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Цена</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Список перевозок</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QTreeWidget" name="MadeUpOrders">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Магазин</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Продукт</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Цена</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>День поставки</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>День выхода срока годности</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>620</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="PreviousDayButton">
      <property name="text">
       <string>Предыдущий день</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="NextDayButton">
      <property name="text">
       <string>Следующий день</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>371</width>
     <height>82</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="Label">
      <property name="text">
       <string>Доход: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="IncomeLabel">
      <property name="text">
       <string>3784</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Выполненные заказы:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="CompletedOrderLabel">
      <property name="text">
       <string>not implemented</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Label_2">
      <property name="text">
       <string>Всего заказов:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="AllQueriesQLabel">
      <property name="text">
       <string>bebra</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
